pages/rops/update/schema/index.js

```diff
@@ -36,10 +36,12 @@ module.exports = req => {
       species: {
         inputType: 'speciesSelector',
         projectSpecies: true,
-        presets: [
-          ...(req.version.data.species || []).filter(s => !s.includes('other')),
-          ...(req.rop.procedures.map(p => p.species))
-        ],
+        presets: (req.version.data.species || []).find(s => s.includes('other'))
+          ? [] // if any "other" type species, don't preselect anything
```

Comment:
I think we still need to preselect the values from `req.rop.procedures.map(p => p.species)` so that soecies which have been previously added to species cannot be removed.
```suggestion
          // don't add species from project if "other" values are selected
          ? req.rop.procedures.map(p => p.species)
```

Reply:
Thanks... missed that.